source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -128,17 +128,154 @@ the Server manual documentation.
    from the Query API expression provided in each example. However,
    both expressions will produce the same aggregation result.
 
+.. important::
+
+   The driver does not provide methods for all aggregation pipeline operators in
+   the Query API. If you need to use an unsupported operation in an
+   aggregation, you can create an expression as a BSON ``Document``
+   type. To learn more about the ``Document`` type, see :ref:`<fundamentals-documents>`.
+
+Array Operations
+~~~~~~~~~~~~~~~~
+
+You can perform an array operation on a value of type ``MqlArray``
+using the methods described in this section.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `all() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#all(java.util.function.Function)>`__
+     - :manual:`$allElementsTrue </reference/operator/aggregation/allElementsTrue/>`
+
+   * - `any() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#any(java.util.function.Function)>`__
```

Comment:
It might be helpful to include an example with any (or all or sum or joinStrings) - I'm not sure if the syntax for these is obvious. For example:

```
totalTicketsSold ... current().<MqlDocument>getArray("showtimes")
    .sum(e -> e.getInteger("ticketsSold"))
```

(I don't think this should replace the existing example.) The syntax is especially odd when the method does not "perform a mapping". For example,

```
current().<MqlInteger>getArray("myNumbers") // [1, 2, 4, 9...]
    .sum(n -> n)
```

Here, the user might expect to be able to write just `.sum()`, but for technical reasons this was not possible to implement. It's not obvious why one would need to write `n -> n`, and it might feel like an annoying trick or surprise. I have trouble coming up with a realistic example of the above (maybe it should just be a note - "The type system requires that certain operations ...require you to pass in an identity function, for example `.sum(n -> n)`").

Reply:
adapted the example. LMK if it is too complicated now

Reply:
Looks good to me. Having a more complex example seems to fit this section. I tried to make it more legible by breaking out some variables.